.github/workflows/test_e2e_py.yml

```diff
@@ -0,0 +1,38 @@
+name: Test end-to-end - Python
```

Comment:
I don't see the tests being removed from the other workflow, so this just looks like a fifth test run